MidasCivil_Adapter/AdapterActions/Execute.cs

```diff
@@ -139,93 +201,118 @@ public bool RunCommand(Open command)
             }
             else
             {
-                if (IsApplicationRunning())
+                m_directory = Path.GetDirectoryName(filePath);
+
+                if (m_midasCivilVersion == "9.5.0.nx")
                 {
-                    Engine.Base.Compute.RecordWarning("MidasCivil process already running");
+                    filePath = filePath.Replace("\\", "\\\\");
+
```

Comment:
Can you add a check here that confirms the file does exist. 

`File.Exists(filepath)`

Currently, there is no feedback if the file path is invalid.